charts/weblate/templates/weblate-ingress.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.global.translationManagementSystemEnabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: weblate
+  namespace: weblate-{{ .Release.Namespace }}
+spec:
+  ingressClassName: nginx
+  rules:
+  {{- if .Values.global.multitenancyEnabled }}
```

Comment:
If this is an adaption and we use the git submodule you could also adapt the domainName inside values.yaml with template rendering. Then we don't have to adapt the submodule.
So you could use tpl function to have a condition for multitenancyEnabled ? ... : ... for example

https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-tpl-function